includes/class-subscription-lists.php

```diff
@@ -0,0 +1,121 @@
+<?php
+/**
+ * Newspack Subscription Lists
+ *
+ * @package Newspack
+ */
+
+namespace Newspack\Newsletter;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Main Newspack Subscription Lists class.
+ *
+ * Subscriptions Lists are Lists which readers can subscribe to. AKA Newsletters.
+ *
+ * Each List is associated with a Audience/List in the Provider and can be associated to one or more tags in the provider
+ */
+class Subscription_Lists {
+
+	/**
+	 * CPT for Newsletter Lists.
+	 */
+	const NEWSPACK_NEWSLETTERS_LIST_CPT = 'newsletter_list';
```

Comment:
We should use a proper plugin prefix for this. Perhaps `newspack_newsletters_list`?

Reply:
:+1: done

Reply:
Actually no, post type names shouldn't be longer than 20 characters

Reply:
The newsletters post type is called `newspack_nl_cpt`. We could borrow the `_nl_` and make `newspack_nl_list`?

Reply:
yep. done